### PR TITLE
Fixes for #59 and #79

### PR DESCRIPTION
**Pull request by glatard**
@johnflavin, you may want to have a look too (for #59).

### CODE REVIEW

File: schema/descriptor.schema.json [hunk 1]
```diff
@@ -324,16 +327,16 @@
...
                 }],
                 "dependencies": {
                     "command-line-flag-separator": ["command-line-flag"],
                     "min-list-entries": ["list"],
                     "max-list-entries": ["list"],
-                    "enum-value-choices": {
+                    "value-choices": {
                         "properties": {
-                            "type": {"enum": ["Enum"]}
+                            "type": {"enum": ["String", "Number"]}
```
Comment (ttaa9) [inline schema/descriptor.schema.json]: You don't want to include `File`?

**Comment by ttaa9** [comment]
Looks good to me. The only issue is that `validator.rb` and `invocationSchemaHandler.py` are not up-to-date with respect to these enum changes.

File: schema/descriptor.schema.json (hunk 1 above)
Comment (glatard) [inline schema/descriptor.schema.json]: Not sure how we could specify a list of values for file... Should it be a list of URLs? Anyway, let's leave it aside for now.

**Comment by glatard** [comment]
@ttaa9 I just updated the two scripts. Please pay particular attention to invocationSchemaHandler.py as I'm a bit unsure of what I did (though it seems to be working with the example).

**Pull request merged**